lib/ts/recipe/session/claimBaseClasses/primitiveClaim.ts

```diff
@@ -27,6 +27,15 @@ export class PrimitiveClaim<T extends JSONValue> extends SessionClaim<T> {
         return res;
     }
 
+    removeFromPayload(payload: any, _userContext?: any): any {
+        const res = {
+            ...payload,
+        };
+        delete res[this.key];
+
+        return res;
+    }
```

Comment:
if this is not being called anywhere, or if you don't think this is useful for the user either, maybe we should remove this function?